app/models/constant.rb

```diff
@@ -0,0 +1,9 @@
+class Constant < ActiveRecord::Base
+  validates_presence_of :name, :value
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.